lib/aem/aemupload.js

```diff
@@ -33,6 +33,7 @@ const { RandomFileAccess } = require("../randomfileaccess");
 const EventEmitter = require("events");
 const UploadError = require("./upload-error");
 const { FilterFailedAssets } = require("../functions/filterfailedassets");
+const { IllegalArgumentError } = require('../error');
```

Comment:
Nitpick: to be consistent with the other imports, single quotes should be turned into double-quotes.
```suggestion
const { IllegalArgumentError } = require("../error");
```

Reply:
Good catch on the quotes, I'll fix those.